src/main/java/org/junitpioneer/jupiter/RetryingTestExtension.java

```diff
@@ -127,11 +138,21 @@ void failed(Throwable exception) {
 					retriesSoFar, maxRetries, minSuccess), exception);
 		}
 
+		private boolean expectedException(Throwable exception) {
+			// if not expected exceptions were specified, all are expected
+			if (expectedExceptions.length == 0)
+				return true;
+
+			return Stream.of(expectedExceptions).anyMatch(type -> type.isInstance(exception));
```

Comment:
Personally, I find `Arrays#stream(...)` more readable as it emphasizes that possibly multiple values are given (even if `expectedExceptions` already uses  plural).